libraries/chain/webassembly/crypto.cpp

```diff
@@ -50,7 +50,7 @@ namespace eosio::chain::webassembly {
          EOS_ASSERT(s.variable_size() <= context.control.configured_subjective_signature_length_limit(),
                     sig_variable_size_limit_exception, "signature variable length component size greater than subjective maximum");
 
-      auto check = fc::crypto::public_key( s, *digest, false );
+      auto check = fc::crypto::public_key( s, *digest, fc::check_canonical_t::no );
```

Comment:
Don't we want to relax this when the protocol feature is enabled? Same for `recover_key`.

Reply:
It was not doing the canonical check even when before the protocol feature.